client/dist/style.css

```diff
@@ -0,0 +1,44 @@
+.app {
+  text-align: center;
+}
+
+.applogo {
+  animation: applogo-spin infinite 20s linear;
+  height 80px;
+}
+
+.appheader {
+  background: #222;
+  height: 150px;
+  padding: 20px;
+  color: white;
+}
+
+.apptitle {
+  font-size: 1.5em;
+}
+
+.appintro {
+  font-size: large;
+}
+
+.@keyframes applogo-spin {
+  from { transform: rotate(0deg); }
+  to { transform: rotate(360deg); }
+}
+
+a {
+  color: #008489;
+  text-decoration: none;
+  background: transparent;
+}
+.room_dls {
```

Comment:
!important should be used sparingly. Please remove or justify.

Reply:
Thanks for the review @rafaelhernandez I'll fix this ASAP.

Reply:
@rafaelhernandez Issue fixed!